packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/SplashImageEditingOptions.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { registerComponent, Components } from '../../../../lib/vulcan-lib/components';
+import { useMulti } from '../../../../lib/crud/withMulti';
+import { defineStyles, useStyles } from '@/components/hooks/useStyles';
+import { PostWithArtGrid } from './PostWithArtGrid';
+import GenerateImagesButton from '@/components/review/GenerateImagesButton';
+
+const styles = defineStyles("SplashImageEditingOptions", (theme: ThemeType) => ({
+  root: { 
+    zIndex: theme.zIndexes.splashHeaderImageOptions,
+    marginBottom: '40px', 
+    gap: '10px',
+    width: 'calc(100vw - 40px)',
+    flexWrap: 'wrap',
+    backgroundColor: theme.palette.background.paper,
+    padding: 20,
+    ...theme.typography.body2,
+  },
+  postWithArtGridContainer: {
+    ...theme.typography.body2,
+    backgroundColor: theme.palette.background.paper,
+  },
+  imageContainer: {
+    width: '200px',
+    height: 'auto',
+    cursor: 'pointer',
+    paddingRight: 5,
+  },
+  image: { 
+    maxWidth: '100%',
+    height: 'auto'
+  },
+}));
+
+export const SplashImageEditingOptions = ({ post }: {
+  post: PostsWithNavigation|PostsWithNavigationAndRevision,
+}) => {
+  const { Loading } = Components;
+  const classes = useStyles(styles);
+
+  const { results: images, loading } = useMulti({
+    collectionName: 'ReviewWinnerArts',
+    fragmentName: 'ReviewWinnerArtImages',
+    terms: {
+      view: 'postArt',
+      postId: post._id,
+      limit: 200,
```

Comment:
is this component only loaded for admins?  bc that's a lot of images

Reply:
yes, and it only shows up for admins once they mouse over a particular button